taskcluster/translations_taskgraph/parameters.py

```diff
@@ -20,10 +20,6 @@ def get_defaults(_):
                 "src": "ru",
                 "trg": "en",
                 "teacher-ensemble": 1,
-                # Used for providing a pretrained backward model. We do not support this yet.
-                "backward-model": "NOT-YET-SUPPORTED",
```

Comment:
I will call out that removing these means that we're diverging further from the snakemake configuration. That's probably OK and necessary for this case - and perhaps a sign that keeping up the pretense of having compatible configs is not worthwhile at this point.

Reply:
Yeah. I went even further and used the kind names in the configuration to simplify the code in the transform

Reply:
I see there's a `taskcluster` key in the config for `Taskcluster-specific pipeline configuration`. Maybe I can put these configuration fields in there.

Reply:
If we're diverging in such big ways I'm not sure we need to worry much about separating the Taskcluster specific things to be honest. @eu9ene - do you have an opinion here?

Reply:
It's fine, I have separated example configs for Snakemake and TC anyway